weave/flow/scorers/json_scorer.py

```diff
@@ -0,0 +1,19 @@
+import json
+from typing import Any
+
+from weave.flow.scorers.base_scorer import Scorer
+
+
+class ValidJSONScorer(Scorer):
+    """Validate whether a string is valid JSON."""
+
+    def score(self, output: Any) -> dict:  # type: ignore
+        try:
+            result = json.loads(output)
+        except json.JSONDecodeError:
+            return {"json_valid": False}
+
+        if isinstance(result, dict) or isinstance(result, list):
```

Comment:
if it loads, isn't it JSON by definition?

Reply:
updated